docs/general/ledger.md

```diff
@@ -23,30 +23,32 @@ for single calls (i.e., bonding and nominating).
 
 :::
 
-The Polkadot [Ledger](https://www.ledger.com/) application is compatible with both the Ledger Nano S
-and the Ledger Nano X devices. Ledger devices are hardware wallets that keep your secret key secured
-on a physical device that does not expose it to your computer or the internet. That is, even if you
-connect your nano via USB to your computer, the private keys will not be leaked. Ledger devices are
-also hierarchical deterministic wallets (HD wallets), that is:
-
-- _Deterministic_ means that there is only one seed phrase to generate all the accounts for
-  different blockchain networks.
+The Polkadot [Ledger](https://www.ledger.com/) application is compatible with the Ledger Nano S and
+the Ledger Nano X devices. Ledger devices are hardware wallets that keep your secret key secured on
+a physical device that does not expose it to your computer or the internet. Even if you connect your
+nano via USB to your computer, the private keys will not be leaked. Ledger devices are also
+hierarchical deterministic wallets (HD wallets), that is:
+
+- _Deterministic_ means that only one seed phrase generates all the accounts for different
+  blockchain networks.
 - _Hierarchical_ means that the accounts are generated in a tree-like structure for different
   purposes.
 
 Ledger devices can be equipped with applications that are blockchain-specific. Such applications are
-usually developed by third parties and they enable the user to transact securely on the blockchain
+usually developed by third parties, and they enable the user to transact securely on the blockchain
 network. The Polkadot Ledger application allows you to manage Polkadot's native token, DOT. It
 supports most of the transaction types of the network, including batch transactions from the Utility
 pallet.
 
 :::note Ledger apps may not support all the transactions
 
 Check [Ledger Polkadot App](https://github.com/Zondax/ledger-polkadot) specification for the list of
-transactions supported. It could be possible that some transactions are supported only on a specific
-version of the app and some transactions are not supported at all by any version. For instance, the
-staking `rebag` extrinsic is supported on the XL version but not on the light version. The
-`setIdentity` extrinsic is not supported by any of the app versions.
+transactions supported. Some transactions are supported only on a specific version of the app, and
+others are not supported at all by any version. For instance, the staking `rebag` extrinsic is
+supported on the XL version but not on the light version. Support for the XL version is limited to
+single calls (not batch calls), meaning you can rebag one account at a time. Also, joining a
+[nomination pool](../learn/learn-nomination-pools.md) is only possible with the XL version. The
+`setIdentity` extrinsic is not supported by any app versions.
```

Comment:
setIdentity works now

https://github.com/Zondax/ledger-polkadot

![image](https://user-images.githubusercontent.com/86818441/218681984-fa9c9a39-a997-449b-be65-085a31e3d429.png)


```suggestion
[nomination pool](../learn/learn-nomination-pools.md) is only possible with the XL version. 
```

Reply:
thanks for the catch!